open-positions/technical-writer.md

```diff
@@ -0,0 +1,79 @@
+---
+tags:
+  - hiring
+  - career
+  - technical
+  - writer
+  - specialist
+title: Technical writer
+date: 2024-02-28
+description: 
+authors:
+  - nikki
+menu: careers
+type: hiring
+hide_frontmatter: false
+hiring: true
+---
+
+## We're looking for Technical Writer to join Dwarves and work remotely.
+
+> **🤘 [Apply now](mailto:spawn@d.foundation)** (We respond within three days)
+
+## Dwarves is research-focused technology firm
+Since 2015, we have helped companies build & ship top-notch software, operate tech teams and invest in ambitious people who are after world's next big things.
```

Comment:
```suggestion
Since 2015, we have helped companies build & ship top-notch products, operate tech teams and invest in ambitious people who are after world's next big things.
```